src/components/pages/RoleSignup/Applications/Styles/test.css

```diff
@@ -0,0 +1,8 @@
+label{
+    background-color: transparent;
+}
+
+/*
+TODO: Find why putting this in mentorApplication.css does not work
+        Why does it work on a separate file?
+*/
```

Comment:
Most likely is that it was being overridden by another file elsewhere that also targets the 'label' identifier. I think @mcfrr3 was hitting a similar issue and found a fix for it. I'd recommend checking with him on it.